File: Data Structures & Algorithms/merge-triplets-to-form-target/submission-6.py
class Solution:
    def mergeTriplets(self, triplets: List[List[int]], target: List[int]) -> bool:
        mina=math.inf
        minb=math.inf
        minc=math.inf
        maxa=0
        maxb=0
        maxc=0
        for i in triplets:
            if i==target:
                return True
            if i[0]<=target[0] and i[1]<=target[1] and i[2]<=target[2]:
                maxa=max(i[0],maxa)
                maxb=max(i[1],maxb)
                maxc=max(i[2],maxc)
                mina=min(i[0],mina)
                minb=min(i[1],minb)
                minc=min(i[2],minc)
        if ( maxa<target[0] or maxb<target[1] or maxc<target[2]):
            return False
        return True

File: Data Structures & Algorithms/min-cost-climbing-stairs/submission-6.java
class Solution {
    public int minCostClimbingStairs(int[] cost) {
        int[] DP=new int[cost.length+1];
        DP[0]=0;
        DP[1]=0;
        for(int i=2;i<DP.length;i++){
            DP[i]=Math.min(cost[i-1]+DP[i-1],cost[i-2]+DP[i-2]);
        }
        return DP[DP.length-1];
    }
}
